src/components/pages/Theme/Theme.jsx

```diff
@@ -261,6 +262,42 @@ export const Theme = () => {
         </Segment>
       </div>
 
+      <div className='section'>
+        <Segment padded>
+          <Header as='h2' dividing>
+            Chip
```

Comment:
I know the design system refers to this as a "Chip", but let's use Semantic terminology and call this "Label." Be sure to reorder it alphabetically.

We can rename the existing "Label" section "Form."